core/services/kraken/main.py

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env python3
+import argparse
+import asyncio
+import logging
+from typing import Any
+
+from commonwealth.utils.apis import GenericErrorHandlingRoute
+from commonwealth.utils.logs import InterceptHandler, get_new_log_path
+from fastapi import FastAPI, status
+from fastapi.responses import HTMLResponse
+from fastapi_versioning import VersionedFastAPI, version
+from loguru import logger
+from pydantic import BaseModel
+from uvicorn import Config, Server
+
+from kraken import Kraken
+
+
+class Extension(BaseModel):
+    name: str
+    tag: str
+    permissions: str
+    enabled: bool
+
+
+SERVICE_NAME = "kraken"
+
+logging.basicConfig(handlers=[InterceptHandler()], level=0)
+
+kraken = Kraken()
+
+try:
+    logger.add(get_new_log_path(SERVICE_NAME))
+except Exception as e:
+    print(f"unable to set logging path: {e}")
+
+
+app = FastAPI(
+    title="Kraken API",
+    description="Kraken is the BlueOS service responsible for installing and managing thirdy-party extensions.",
+)
+app.router.route_class = GenericErrorHandlingRoute
+logger.info("Releasing the Kraken!")
+
+
+@app.get("/extensions_manifest", status_code=status.HTTP_200_OK)
+# @temporary_cache(timeout_seconds=300)
```

Comment:
Lost code

Reply:
You can try: https://github.com/long2ice/fastapi-cache

Reply:
it turned out there is a lot of overhead to this approach. I'll get a cache in there later, it is not that important